packages/flutter_test/lib/src/test_pointer.dart

```diff
@@ -162,16 +172,24 @@ class TestGesture {
   final TestPointer _pointer;
 
   /// Send a move event moving the pointer by the given offset.
-  Future<void> moveBy(Offset offset, { Duration timeStamp = Duration.zero }) {
+  Future<void> moveBy(Offset offset, { Duration timeStamp = Duration.zero, double pressure = 0.0,
+                                                                           double pressureMin = 0.0,
+                                                                           double pressureMax = 1.0 }) {
     assert(_pointer._isDown);
-    return moveTo(_pointer.location + offset, timeStamp: timeStamp);
+    return moveTo(_pointer.location + offset, timeStamp: timeStamp, pressure: pressure,
+                                                                    pressureMin: pressureMin,
+                                                                    pressureMax: pressureMax);
   }
 
   /// Send a move event moving the pointer to the given location.
-  Future<void> moveTo(Offset location, { Duration timeStamp = Duration.zero }) {
+  Future<void> moveTo(Offset location, { Duration timeStamp = Duration.zero, double pressure = 0.0,
+                                                                             double pressureMin = 0.0,
+                                                                             double pressureMax = 1.0 }) {
     return TestAsyncUtils.guard<void>(() {
       assert(_pointer._isDown);
-      return _dispatcher(_pointer.move(location, timeStamp: timeStamp), _result);
+      return _dispatcher(_pointer.move(location, timeStamp: timeStamp, pressure: pressure,
+                                                                       pressureMin: pressureMin,
+                                                                       pressureMax: pressureMax), _result);
     });
   }
 
```

Comment:
i think we should probably not add the pressure arguments to the TestPointer and WidgetController APIs, because that will quickly get out of hand when we add everything else also.

Better instead to add a new method to TestPointer that takes an arbitrary PointerEvent, and updates isDown accordingly, and returns it, and then a new pair of methods to TestGesture, one to create a gesture from a custom PointerDownEvent and one to send the next PointerEvent. (The existing methods can all be refactored in terms of these more generic methods too.)

That way these APIs will handle all future API changes too without having to keep growing to match the underlying pointer API.